plugins/defaultplugins/defaultplugins_init.go

```diff
@@ -40,14 +43,23 @@ import (
 	"github.com/ligato/vpp-agent/plugins/govppmux"
 	"github.com/ligato/vpp-agent/plugins/linuxplugin"
 	ifaceidx2 "github.com/ligato/vpp-agent/plugins/linuxplugin/ifaceidx"
+	"github.com/ligato/cn-infra/datasync/kvdbsync"
 )
 
 // Plugin implements Plugin interface, therefore it can be loaded with other plugins
 type Plugin struct {
-	Transport    datasync.TransportAdapter `inject:""`
-	ServiceLabel *servicelabel.Plugin
+	Deps		 							// inject
+	Transport    datasync.KeyProtoValWriter // inject
+	Watch    datasync.KeyValProtoWatcher // inject
+
+	logger       logging.PluginLogger
```

Comment:
Please remove all following fields (all of them we have in  anonoymous composistion Deps by reusing cn-infra localdeps/*)